sqldelight-compiler/src/test/view-interface-fixtures/only-table-type/output/com/sample/Person.kt

```diff
@@ -14,7 +13,7 @@ public data class Person(
   public val last_name: String?,
   public val is_cool: Boolean,
   public val friends: List<Person>?,
-  public val shhh_its_secret: @Redacted String
+  public val shhh_its_secret: String
```

Comment:
Now the annotation is lost. Would not it be possible to generate:

```suggestion
  @Redacted public val shhh_its_secret: String
```

And then the annotation syntax in the sql file needs to be vaild. In my case, I'd need to declare a message on `@Deprecated`

Reply:
Yes, it's a one-line change but wouldn't that be different in semantics?